back-end/apps/api/src/users/dtos/version-check-response.dto.ts

```diff
@@ -8,4 +8,42 @@ export class VersionCheckResponseDto {
   })
   @Expose()
   success: boolean;
+
+  @ApiProperty({
+    description: 'The latest supported frontend version (from server configuration)',
+    example: '1.2.0',
+    nullable: true,
+  })
+  @Expose()
+  latestSupportedVersion: string | null;
+
+  @ApiProperty({
+    description: 'The minimum supported frontend version (from server configuration)',
+    example: '1.0.0',
+    nullable: true,
+  })
+  @Expose()
+  minimumSupportedVersion: string | null;
+
+  @ApiProperty({
+    description: 'URL to the releases page where the user can download updates',
+    example: 'https://github.com/hashgraph/hedera-transaction-tool/releases/tag/v1.2.0',
+    nullable: true,
+  })
+  @Expose()
+  updateUrl: string | null;
+
+  @ApiProperty({
+    description: 'True if the user version is older than the latest supported version',
+    example: true,
+  })
+  @Expose()
+  updateAvailable: boolean;
```

Comment:
If updateUrl is null, then the updateAvailable should be false. Is there any other reason to have this property?

Reply:
yes, you are right, the front end will check the `updateUrl` in order to decide if an update is needed